src/test/java/io/vertx/ext/embeddedmongo/test/EmbeddedMongoVerticleTest.java

```diff
@@ -31,7 +31,7 @@
   @Override
   public VertxOptions getOptions() {
     // It can take some time to download the first time!
-    return new VertxOptions().setMaxWorkerExecuteTime(30 * 60 * 1000);
+    return new VertxOptions().setMaxWorkerExecuteTime(30 * 60 * 1000).setWarningExceptionTime(30 * 60 * 1000);
```

Comment:
could extract the 30 * 60 * 1000 to a const

Reply:
the change wasn't working to suppress the warnings anyway so I'm removing it.